spec/rails_helper.rb

```diff
@@ -58,6 +58,10 @@
     DatabaseCleaner.clean
   end
 
+  config.before(:each) do
```

Comment:
I'm a little worried about the speed implication of this... but probably more worried about the implications of it not being the case for all these tests! 

Big fan of this being its own commit, nice one.